### PR TITLE
Bump webrtc-adapter to 7.7.0

### PR DESCRIPTION
**Pull request by keianhzo**
Bump webrtc-adapter to 7.7.0

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -27672,8 +27672,8 @@
       "dev": true
     },
     "networked-aframe": {
-      "version": "github:mozillareality/networked-aframe#ae8ca30fcded2369c8702686b07cce46dfc16306",
-      "from": "github:mozillareality/networked-aframe#master",
+      "version": "github:mozillareality/networked-aframe#a1f1ec48caacadd6e4fcf13f413509d3bfe1aef6",
+      "from": "github:mozillareality/networked-aframe#fix/remote-clients-sending-invalid-data-cherry-picked",
```
Comment (brianpeiris) [inline package-lock.json:27676]: This was unintentionally reverted.

**Comment by brianpeiris** [comment]
I also think we should merge this into `master` now so that it can get some real-world testing on HMC.

**Comment by keianhzo** [comment]
@robertlong @Brian This should be ok now.

**Comment by keianhzo** [comment]
@brianpeiris https://github.com/mozilla/hubs/pull/3686

**Pull request merged**